server/app/views/admin/programs/ProgramBlockAllAspectsView.java

```diff
@@ -49,45 +53,61 @@
 import views.style.StyleUtils;
 
 /**
- * Renders a page for an admin to edit the configuration for a single block of a program.
+ * Renders a page for an admin to view or edit the configuration for a single
+ * block of an active or draft program.
  *
- * <p>Contains elements to:
+ * <p>For drafts editing this contains elements to:
  *
  * <ul>
  *   <li>Delete the block
  *   <li>Edit the name and description
  *   <li>View, add, delete and reorder questions
  *   <li>View and navigate to the visibility criteria
  * </ul>
+ *
+ * For Active program viewing it contains the same elements, but UI elements
+ * that can be used for editing.
  */
-public final class ProgramBlockEditView extends ProgramBlockView {
+public final class ProgramBlockAllAspectsView extends ProgramBlockView {
```

Comment:
Sounds good and yes I was trying to find a name that would work and not clash. AbstractProgramBlockView was another option I considered for the parent, but "Base" sounds better. I'm creating an issue that tracks the cleanups (there were a few others that I also wanted to track)

Reply:
Unresolving so we remember to do it in this PR

Reply:
Ok, sorry, I misunderstood you, I thought you meant that we merge and submit and rename as a separate change.

Reply:
Let's do it in this PR rather than having churn in the permanent repo.

By squared away, I meant resolving the discussions and have approval otherwise so that the incremental change history wasn't lost.